gpytorch/lazy/block_diag_lazy_tensor.py

```diff
@@ -115,6 +115,18 @@ def inv_quad_logdet(self, inv_quad_rhs=None, logdet=False, reduce_inv_quad=True)
             logdet_res = logdet_res.view(*logdet_res.shape).sum(-1)
         return inv_quad_res, logdet_res
 
+    def matmul(self, other):
+        from .diag_lazy_tensor import DiagLazyTensor
```

Comment:
Why isn't this imported at the top of the file? Does it cause a circular import or something?

Reply:
Yes, since `BlockDiagLazyTensor` is imported top-of-file in `diag_lazy_tensor.py` (see below).